train.py

```diff
@@ -89,13 +90,13 @@ def forward(self, imgs, annotations, obj_list=None):
 def train(opt):
     params = Params(f'projects/{opt.project}.yml')
 
-    if params.num_gpus == 0:
```

Comment:
actually, this part is meant to train on cpu if someone wants to debug the code.
Since `torch.cuda.is_available()` always returns True on cuda servers, you can't debug the training easily.